package/samplers/auto_sampler/tests/test_auto_sampler.py

```diff
@@ -166,3 +168,7 @@ def test_multi_thread() -> None:
     assert "RandomSampler" in sampler_names
     assert "GPSampler" in sampler_names
     assert "CmaEsSampler" in sampler_names
+
+
+def test_picklize() -> None:
+    pickle.dumps(AutoSampler())
```

Comment:
How about test if we can load the pickled sampler?

```suggestion
    pickle.loads(pickle.dumps(AutoSampler()))
```

Reply:
SGTM!